setup.cfg

```diff
@@ -15,6 +15,7 @@ line_length=88
 warn_return_any = True
 warn_unused_configs = True
 ignore_missing_imports = True
+disable_error_code = override
```

Comment:
need to address at a later time, but some mixin / abstraction issues with mypy

Reply:
We should document this if it is needed in the future